### PR TITLE
Make the lambda timeout length configurable

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
Also catch the lambda timeout exception and return message in controller
response.

### CODE REVIEW

File: src/test/java/gov/usgs/aqcu/controller/LambdaReportControllerTest.java [hunk 1]
```diff
@@ -92,6 +93,21 @@ public void getReportLambdaErrorTest2() {
         assertEquals(404, result.getStatusCode().value());
         assertTrue(result.getBody().contains("not found"));
     }
+    
+    @Test
+    public void getReportLambdaTimeoutErrorTest() {
```
Comment (cschroed-usgs) [inline src/test/java/gov/usgs/aqcu/controller/LambdaReportControllerTest.java:98]: I love seeing tests for error handling 👍 🍰

**Review by cschroed-usgs — APPROVED** [review]
Smart change! Merge when ready!

**Review by zmoore-usgs — CHANGES_REQUESTED** [review]
So the existing 18000 values were actually already configurable via environment variables thanks to the Spring fuzzy environment variables matching, in the same way that the names are configurable. If you look at the config.env for the Test-tier Gateway in S3 you'll see:

```
LAMBDA_FUNCTIONS_GWVISITREVIEWSTATUS_NAME=NWISRALambda-TEST-GWVRSTATREPORT
LAMBDA_FUNCTIONS_GWVISITEXTREMES_NAME=NWISRALambda-TEST-GWVEXTREPORT
```

To override the default 18000 timeout you can just add overrides to that config.env like so:

```
LAMBDA_FUNCTIONS_GWVISITREVIEWSTATUS_TIMEOUT=30000
LAMBDA_FUNCTIONS_GWVISITEXTREMES_TIMEOUT=60000
```

So having the wrapper variables is not actually necessary. Other changes with better error handling and associated test look good though!

**Comment by kmschoep-usgs** [comment]
The wonders and magic of Spring never ceases to amaze me.   I'll revert the config changes.

**Review by zmoore-usgs — APPROVED** [review]

**Comment by cschroed-usgs** [comment]
Oh yeah! I forgot about the fuzzy env var matching feature. Nice simplification!

**Pull request merged**